tests/board_selection_test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import mock
+from mock import patch
+from mock import Mock
+
+from ..lib.board_selection import BoardSelection
+from ..lib.ui import UI
+from ..lib.game_selection import *
+from ..lib.tictactoe_board import *
+
+
+class TestBoardSelection:
```

Comment:
Can we have tests here that returns `TicTacToeBoard`s of the correct size?

Reply:
Done!

Reply:
Now with the addition of the invalid board size message, could we add a test for that as well? 😎